examples/postgresql/postgres-blueprint.yaml

```diff
@@ -70,7 +70,7 @@ actions:
       name: deleteDump
       args:
         image: ghcr.io/kanisterio/postgres-kanister-tools:0.84.0
-        namespace: "{{ .Namespace.Name }}"
+        namespace: "{{ .StatefulSet.Namespace }}"
```

Comment:
Why is this required? Note that downstream [adopters](https://github.com/kanisterio/kanister/blob/master/ADOPTERS.md) of Kanister use Namespace as the subject when executing `delete` action and this change will break them. I would recommend reverting this

Reply:
I was getting nil pointer reference for `Namespace.Name`. if downstream projects are using `Namespace.Name`, it should be consistent across phases right? we only have `Namespace.Name` for delete phase here but other 2 phases have `StatefulSet.Namespace`

Reply:
reverted the change